Components/Modal/modal.css

```diff
@@ -0,0 +1,175 @@
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
+
+* {
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
+
+:root {
+    /* common modal css variables */
+    --modal-width: 22rem;
+    --modal-bg-color: #bae6fd;
+    --modal-border: 1px solid #000;
+    --modal-border-radius: 0.5em;
+    --modal-button-color: #000;
+    --modal-button-hover-color: #7dd3fc;
+    --modal-button-hover-border-radius: 0.4em;
+    /* simple modal css variables */
+    --simple-modal-height: 12rem;
+    --simple-modal-icon-color: #525252;
+    --simple-modal-icon-size: 3rem;
+    --simple-modal-divider-border: 1px solid #000;
+    /* confirmation modal css variables */
+    --confirmation-modal-height: 12rem;
+    --confirmation-modal-divider-border: 1px solid #000;
+    /* focus color is seen when you use tab key to access buttons or links */
+    --modal-button-focus-outline-color: #000;
+}
+
+.modal {
+    width: var(--modal-width);
+    background-color: var(--modal-bg-color);
+    border: var(--modal-border);
+    border-radius: var(--modal-border-radius);
+
+    /* flex - properties */
+    display: flex;
+    flex-direction: column;
+    gap: 1em;
+    align-items: center;
+}
+
+.modal.simple-modal {
+    gap: 0;
```

Comment:
Looked into the deploy preview & did not see gap 0 making any significant change. Why was this required?

Reply:
Actually it does make a small difference in simple modal when u scroll there is a gap that doesn't look good, u can try using inspect to see the difference